index.js

```diff
@@ -4,6 +4,10 @@ if (typeof process !== 'undefined' && parseInt(process.versions.node.split('.')[
   process.exit(1)
 }
 
-module.exports.helloWorld = function () {
-  console.log('Hello world !')
+module.exports = {
+  Tokens: require('./src/Tokens'),
+  Authflow: require('./src/MicrosoftAuthFlow'),
+
+  Authenticate: require('./src/MicrosoftAuthentication'),
```

Comment:
is it a typo to have two of these with the change in caps?

Reply:
No, my intention was to solve my OCD and have one capitalized for when I use it, However, I have since fixed that problem. I am going to just keep the lower case one.